### ISSUE TITLE
Npm version outdated

### ISSUE THREAD

**Issue by t27** [edwardhotchkiss/short#41]
npm shows the older 2.5.2 version, could you please update the package on npm so that using the newer 2.6.0 version is easy

**Comment by edwardhotchkiss** [comment]
Feel free to PR

**Comment by t27** [comment]
Let me rephrase, 
The version of short on npm (https://www.npmjs.com/package/short) 
Is 2.5.2, whereas the latest version in this repo's package json is 2.6.0,

It needs to be updated on npmjs.com

I don't think it is something that can be fixed through a PR, correct me if I am wrong

**Comment by t27** [comment]
Please can you run `npm publish` with your npmjs.com account? or add me as a collaborator on npm so that i can publish it
https://docs.npmjs.com/cli/owner
my npm account - https://www.npmjs.com/~tarangshah

Thanks

**Comment by edwardhotchkiss** [comment]
Published.

**Comment by t27** [comment]
Thank you so much :)

**Comment by t27** [comment]
Was stuck on an issue which urgently needed this npm update, thanks :D
